### ISSUE TITLE
Bug The workspace is not saved correctly

### ISSUE THREAD

**Issue by d3v0m4r** [vim-ctrlspace/vim-ctrlspace#294]
I use nvim 0.5

When you have more than one tab with open buffers, they are not saved correctly, the buffers inside the first tab are saved correctly, but the buffers of the second tab are not saved, only a buffer appears in the second tab

The names that are given to each tab in the workspace are not saved well either

**Comment by Konfekt** [comment]
This sounds just like https://github.com/vim-ctrlspace/vim-ctrlspace/issues/293 that was just solved?

**Comment by jyscao** [comment]
I ran into this same issue myself the other day, and I am using Neovim 0.5 as well. So perhaps it is related to that.

I will take a look into this over the next couple days.

**Comment by Konfekt** [comment]
@gregoirehenry Does it work for you? @d3v0m4r Does `echo has('nvim-0.5')` show `1` (for https://github.com/vim-ctrlspace/vim-ctrlspace/commit/d636f7d56a7894fc80d12e5933aadd046b9eb226 to apply)?

**Comment by GregoireHENRY** [comment]
@Konfekt since last time we discussed (#293) everything is working perfectly on my side :)

**Comment by jyscao** [comment]
@Konfekt on my Neovim 0.5 (from daily build Ubuntu PPA), `echo has('nvim-0.5')` is `1`, while `(has('patch-8.1.149')` is `0`

**Comment by Konfekt** [comment]
Strange. So why does it break on the same Vim on your system but work on @GregoireHENRY ?

Might it be that the  worked space has been saved before by a version of Nvim < 0.5 ?

**Comment by jyscao** [comment]
That shouldn't be the case, since I've been using 0.5 since before the new year.

Also the data and commands inside of the project's `.git/cs_workspaces` file look sensible.

Anyway, guess I'll keep digging.

**Comment by Konfekt** [comment]
@d3v0m4r , @jyscao : Problem could be that `ctrlspace#workspaces#SaveWorkspace(name)` only saves the workspace correctly in Nvim 0.5 since the last commit, but your `.git/cs_workspaces` was saved before?

**Comment by jyscao** [comment]
@Konfekt yes, you're probably right. The newer workspaces I've since created are working good again. If it's working good for @d3v0m4r too, then we can close this issue.
